dir.props

```diff
@@ -21,13 +21,11 @@
   <Import Project="build/BuildDefaults.props" />
   <Import Project="build/BuildOutputs.props" />
   <Import Project="build/Stage0.props" />
+  <Import Project="build/ToolInfo.props" />
   <Import Project="build/VersionBadge.props" />
   <Import Project="build/Microsoft.DotNet.Cli.BundledRuntimes.props" />
   
   <PropertyGroup>
-    <NuGetPackagesDir>$(NUGET_PACKAGES)</NuGetPackagesDir>
```

Comment:
Yea, I'd undo this change [NuGetPackagesDir]. It's so central and impactful, I think it belongs where it was.